src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/AbstractApiAction.java

```diff
@@ -581,4 +583,11 @@ protected boolean isReadOnlyAndAccessible(Settings settings, String name) {
 		return true;
 	}
 
+	protected boolean isHiddenAndAccessible(Settings settings, String name) {
```

Comment:
Something does not look correct here.
Method is "isHiddenAndAccessible" but it returns "false" when object is hidden and not super admin. 
This is  too confusing.

Can we name it instead  "isAccessible" or convert it to opposite "isInaccessible" or any other name that is less confusing.